hazelcast-integration/spring-hibernate-2ndlevel-cache/README.md

```diff
@@ -29,13 +30,13 @@ In order to configure Hazelcast as second-level cache provider, you need to add
 <dependency>
 	<groupId>com.hazelcast</groupId>
 	<artifactId>hazelcast-hibernate53</artifactId>
-	<version>2.0.0</version>
+	<version>2.2.2-SNAPSHOT</version>
 </dependency
 
 <dependency>
 	<groupId>com.hazelcast</groupId>
 	<artifactId>hazelcast</artifactId>
-	<version>4.0.1</version>
+	<version>5.3.0-SNAPSHOT</version>
```

Comment:
5.2.0 instead?

Reply:
In other code samples we use SNAPSHOT versions so I've followed this approach, we encourage users to test new features within code samples ;)